.travis.yml

```diff
@@ -17,6 +19,10 @@ before_script:
   > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
+- curl -L https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/bin/utils/openapi-generator-cli.sh
+  > ./openapi-generator-cli
+- chmod +x ./openapi-generator-cli
+- "./openapi-generator-cli validate -i public/doc/openapi-3-v1.2.json"
```

Comment:
@bzwei @mkanoor Does it make sense to expand this so it dynamically tests all available versions?  Having a hard-coded version just means we will have to remember to update reach time it changes and will likely mean we miss capturing errors early.

Reply:
yes. let's simply validate all versions under this folder.